build/hotfix.sh

```
@@ -2,10 +2,10 @@
 
 # This script is used to apply hotfixes.
 
-set -eu
+set -euo pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)